consul/init.sls

```diff
@@ -1,11 +1,2 @@
-{% from "consul/map.jinja" import consul with context %}
-
-consul:
-  pkg.installed:
-    - pkgs: {{ consul.pkgs }}
-  service:
-    - running
-    - name: {{ consul.service }}
-    - enable: True
-    - require:
-      - pkg: consul
+include:
+  - .service
```

Comment:
As mentioned in the comment on the `service` state, the service shouldn't be the entry point into the formula. You can either keep this file as a bare `include` of other states, or you can move the contents of `install.sls` into this file.